charts/kyverno-policies/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: kyverno-policies
-version: v2.5.5
+version: v2.5.6
```

Comment:
Currently the Helm charts are released via release branches. I will bump versions while releasing, can we revert this?

Reply:
Done